account_billing/views/account_billing_views.xml

```diff
@@ -0,0 +1,251 @@
+        <field name="model">account.billing</field>
+        <field name="arch" type="xml">
+            <search string="Search Billings">
+                <field name="name" string="Name" />
+                <field name="partner_id" />
+                <filter
+                    name="state_draft"
+                    string="Draft"
+                    domain="[('state','=','draft')]"
+                    help="Draft Billing"
+                />
+                <filter
+                    name="state_billed"
+                    string="Billed"
+                    domain="[('state','=','billed')]"
+                    help="Billed Billing"
+                />
+                <separator />
+                <filter
+                    string="Vendor"
+                    domain="[('bill_type','=','in_invoice')]"
+                    name="vendor_filter"
+                />
+                <filter
+                    string="Customer"
+                    domain="[('bill_type','=','out_invoice')]"
+                    name="customer_filter"
+                />
+                <group expand="0" string="Group By">
+                    <filter
+                        name="group_by_customer"
+                        string="Customer"
+                        context="{'group_by':'partner_id'}"
+                    />
+                    <filter
+                        name="group_by_state"
+                        string="Status"
+                        context="{'group_by':'state'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+    <record id="view_account_billing_form" model="ir.ui.view">
+        <field name="name">account.billing.form</field>
+        <field name="model">account.billing</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <button
+                        name="validate_billing"
+                        class="oe_highlight"
+                        states="draft"
+                        string="Validate"
+                        type="object"
+                    />
+                    <button
+                        name="action_cancel"
+                        states="draft"
+                        string="Cancel"
+                        type="object"
+                    />
+                    <button
+                        name="action_cancel"
+                        states="billed"
+                        string="Cancel"
+                        type="object"
+                        confirm="Are you sure to cancel this document?"
+                    />
+                    <button
+                        name="action_cancel_draft"
+                        states="cancel"
+                        string="Set to Draft"
+                        type="object"
+                    />
+                    <field
+                        name="state"
```

Comment:
Use badges for states?

                <field name="state" widget="badge" decoration-success="state == 'billed'" decoration-info="state == 'draft'" decoration-danger="state == 'cancel'"/>